File: android-sdk/src/main/java/com/checkout/android_sdk/Utils/Environment.java
package com.checkout.android_sdk.Utils;

public enum Environment {
    SANDBOX(
            "https://api.sandbox.checkout.com/tokens",
            "https://api.sandbox.checkout.com/.well-known/content-encoding/jwks",
            "https://api.sandbox.checkout.com/tokens"
    ),
    LIVE(
            "https://api.checkout.com/tokens",
            "https://api.checkout.com/.well-known/content-encoding/jwks",
            "https://api.checkout.com/tokens"
    );

    public final String token;
    public final String jwks;
    public final String googlePay;

    Environment(String token, String jwks, String googlePay) {
        this.token = token;
        this.jwks = jwks;
        this.googlePay = googlePay;
    }
}

File: sdk/src/main/java/com/checkout/sdk/utils/Environment.kt
package com.checkout.sdk.utils

enum class Environment(
    val host: String
) {
    SANDBOX("https://sandbox.checkout.com/api2/"),
    LIVE("https://api2.checkout.com/");

    companion object {
        const val TOKEN_PATH = "v2/tokens/card"
        const val GOOGLE_PAY_PATH = "tokens"
    }
}
